src/openklant/templates/index.html

```diff
@@ -1,51 +1,46 @@
-{% extends "main.html" %}
+{% extends "maykin_common/api/index_base.html" %}
+{% load i18n %}
 
 {% block content %}
```

Comment:
are there any extra blocks that maykin common can/should define to remove more boilerplate from this template? The ambition is really that all API projects can simply stick to overriding the blocks defined here: https://maykin-django-common.readthedocs.io/en/latest/apis.html#maykin-common-api-index-base-html